uui/components/pickers/PickerToggler.tsx

```diff
@@ -53,6 +53,9 @@ function PickerTogglerComponent<TItem extends string, TId>(props: PickerTogglerP
 
     const renderItem = (row: DataRowProps<TItem, TId>) => (
         <Tag
+            rawProps={ {
+                'data-testid': `uui-PickerToggler-item-${row.rowKey}`,
```

Comment:
as far as I remember we agreed to use semantic markup for tags, i.e. add aria-role to the wrapper div and to each Tag item. Adding "data-testid" is less preferrable.

Reply:
See the comment before. A see it as an antipattern. And we agreed on adding data-testids for the cases, where multiple items can be added.

Reply:
In this specific case we have an opportunity to improve a11y by using semantic markup (aria roles for list & items https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role)